PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/resources/META-INF/azure-intellij-plugin-azure-sdk.xml

```diff
@@ -171,41 +171,16 @@
 
         <localInspection
                 language="JAVA"
-                shortName="UpdateCheckpointAsyncSubscribeChecker"
-                displayName="Update Checkpoint Async Subscribe Checker"
+                shortName="UseOfBlockOnAsyncClientsCheck"
+                displayName="Use Of Block On Async Clients Check"
                 groupName="Azure SDK Rules"
                 enabledByDefault="true"
                 level="WARNING"
-                implementationClass="com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool.UpdateCheckpointAsyncSubscribeChecker">
+                implementationClass="com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool.UseOfBlockOnAsyncClientsCheck">
 
-            <class>
-                com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool.UpdateCheckpointAsyncSubscribeChecker
-            </class>
-        </localInspection>
-
-        <localInspection
```

Comment:
Should this be removed?

Reply:
no this was a merge conflict error thanks for the flag😅